corehq/apps/app_manager/migrations/0025_conditional_case_update_deleted_apps.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management import call_command
+from django.db import migrations
+
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _migrate_to_conditional_case_update(apps, schema_editor):
+    call_command("migrate_to_conditional_case_update", "--deleted-apps-only", "--only-run-once")
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('app_manager', '0024_applicationreleaselog_info'),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            _migrate_to_conditional_case_update,
+            reverse_code=migrations.RunPython.noop,
+        ),
+    ]
```

Comment:
If it makes it simpler you can put the migration into a separate PR so that you don't have to worry about merging in the other changes.